installation/configuration.md

```diff
@@ -120,77 +141,32 @@ rewrite ^/([^/\.]+)/([^/\.]+)/?$ /index.php?act=$1.$2;
 rewrite ^/([^/\.]+)/([^/\.]+)/([^/\.]+)/?$ /index.php?act=$1.$2&id=$3;
 ```
 
-This and by desabling the 
+## Theme Configuration
+
+Leantime allows you to white label the entire system by changing the color scheme of your site, set your own logo, and also set primary and secondary color. This can be easily done by updating the `config/.env` file:
```

Comment:
Leantime *can be white labeled through the use of theme creation.

@marcelfolaron can you chime in here on the right wording? You could update the code but without creating a theme to re-add, your changes will be overwrote if you update the system. This should be highlighted.

Reply:
Yeah let's reword that to:

_Leantime allows you to customize the color scheme and logo by setting the corresponding values in your environment file_